src/main/java/org/apache/cassandra/sidecar/cdc/CassandraConfig.java

```diff
@@ -0,0 +1,68 @@
+package org.apache.cassandra.sidecar.cdc;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.function.Supplier;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import org.apache.cassandra.config.Config;
+import org.apache.cassandra.config.DatabaseDescriptor;
+import org.apache.cassandra.config.YamlConfigurationLoader;
+import org.apache.cassandra.sidecar.Configuration;
+
+/**
+ * Custom Cassandra configurator
+ */
+@Singleton
+public class CassandraConfig implements Supplier<Config>
+{
+    private Config config;
+
+    @Inject
+    public CassandraConfig(Configuration config)
```

Comment:
I think the input can just be a string mean the cassandra config file path, and can also be easilily inject